### ISSUE TITLE
Add syntax highlight in the output

### ISSUE THREAD

**Issue by doggy8088** [msyrus/vscode-go-doc#2]
Is it possible to add some colors in the output? Or is there any extension can help with that?

![image](https://user-images.githubusercontent.com/88981/88872390-9fe2a680-d24c-11ea-93e6-fb532c7fd63c.png)

Some of the function contains long documentations. If there are some colors, that will improve readability.

**Comment by msyrus** [comment]
Yes, that will be great. If you can manage to create a PR, I will be happy to review and merge it.

**Comment by msyrus** [comment]
After doing some search, I have found that vscode doesn't support colorization for specific output channel [here](https://github.com/Microsoft/vscode/issues/19561). As a result there is a chance to conflict with other extensions that also colors output. 

I would suggest to implement a configurable option that prints the definition in a side editor and apply go language theme
